source/com.google.android.gms/play-services-safetynet/Additions/Additions.cs

```diff
@@ -29,9 +29,12 @@ public static async Task<bool> ValidateWithGoogle (this SafetyNetApiAttestationR
 
             response.EnsureSuccessStatusCode ();
 
-            var json = JsonObject.Parse (data);
-            if (json.ContainsKey ("isValidSignature"))
-                return json ["isValidSignature"].ToString ().Trim ('"').Equals ("true");
+            JsonDocument doc = JsonDocument.Parse(data);
```

Comment:
Since you're improving this, maybe remove:
```csharp
var data = await response.Content.ReadAsStringAsync ();
```

And instead do something like:
```csharp
using var stream = await response.Content.ReadAsStreamAsync();
JsonDocument doc = await JsonDocument.ParseAsync(stream);
```
Just so it doesn't create an intermediate `string` of the .json document.

Reply:
Fixed/improved.

I was merely investigating the new API, because I was/am not familiar with it and seems I did not turn on C# 8.0 and newer features in dotnetfiddle...